packages/stryker-api/src/report/Reporter.ts

```diff
@@ -38,9 +39,15 @@ interface Reporter {
   onAllMutantsTested?(results: MutantResult[]): void;
 
   /**
-   * Called when stryker wants to quite
+   * Called when the mutation score is calculated.
+   * @param score The immutable structured score result (tree)
+   */
+  onScore?(score: ScoreResult): void;
```

Comment:
As discussed offline, we should rename it to `onScoreResult`  or `onScoreCalculated`.